typing/types.mli

```diff
@@ -565,6 +565,7 @@ type class_type_declaration =
   { clty_params: type_expr list;
     clty_type: class_type;
     clty_path: Path.t;
+    clty_ty: type_declaration;
```

Comment:
Could we have a documentation comment that explains what this type declaration is?

Florian suggests "the type of the open row in this class type, used in `#t`" (if I understood correctly)

Reply:
Will https://github.com/ocaml/ocaml/pull/11569/commits/cebcd2698ffd3f4321ac9486a652fad95339cf1c suffice?

Reply:
I believe so. Thanks!